packages/tree-widget/src/components/header/TreeHeader.tsx

```diff
@@ -0,0 +1,60 @@
+import * as React from "react";
+import { IconButton } from "../IconButton";
+import { SearchBar } from "../search-bar/SearchBar";
+import { SearchOptions } from "../TreeFilteringState";
+import { TreeWidget } from "../../TreeWidget";
+import "./TreeHeader.scss";
+
+export interface TreeHeaderComponentProps {
+  showAll?: () => void;
+  hideAll?: () => void;
+  invert?: () => void;
+  searchOptions: SearchOptions;
+}
+
+export function TreeHeaderComponent(props: TreeHeaderComponentProps) {
+  const { searchOptions } = props;
```

Comment:
nit pick, you can destructure all the props, and might as well move it above like so 

```tsx
export function TreeHeaderComponent({ searchOptions, showAll, hideAll, searchOptions }: TreeHeaderComponentProps) {
```

Reply:
Also, should this be a patch or a new minor bump? Since you're exporting a new component maybe its a minor bump? I'm not too bothered with how it is tho.

Reply:
Not a fan of that myself, but sure. :)